int32 message_id   # Id for CAN Message
int32[] message_data   # Data for particular CAN Message
